pages/library/index.js

```diff
@@ -0,0 +1,11 @@
+import Background from "@/components/common/Background/blue.js";
```

Comment:
You can make the Background component to variant.
This makes your component reuseable again. 
later on you just need to call it similar to this.
```js
<Background variant={blue} />
```